pkgs/servers/x11/xorg/xwayland.nix

```diff
@@ -1,37 +1,31 @@
 { lib, wayland, wayland-protocols, xorgserver, xkbcomp, xkeyboard_config
-, epoxy, libxslt, libunwind, makeWrapper, egl-wayland
-, defaultFontPath ? "" }:
-
-with lib;
+, epoxy, libtirpc, libxslt, libunwind, makeWrapper, egl-wayland, fetchurl, meson, ninja
+, pkg-config, fontutil, defaultFontPath ? "" }:
 
 xorgserver.overrideAttrs (oldAttrs: {
 
-  name = "xwayland-${xorgserver.version}";
-  buildInputs = oldAttrs.buildInputs ++ [ egl-wayland ];
+  name = "xwayland-21.1.0";
+  src = fetchurl {
+    url = "https://xorg.freedesktop.org/archive/individual/xserver/xwayland-21.1.0.tar.xz";
+    sha256 = "sha256-inHD6LlcQ7uRvD16T/kHVFagz+KXch28jUp2zyQdgvs=";
+  };
+  patches = [ ];
```

Comment:
```suggestion

```

Reply:
Required to remove the `patches` from the overlay.